packages/lib/BaseApplication.ts

```diff
@@ -854,6 +854,8 @@ export default class BaseApplication {
 				});
 		}
 
+		setDateFormat(Setting.value('dateFormat'));
+		setTimeFormat(Setting.value('timeFormat'));
 		time.setDateFormat(Setting.value('dateFormat'));
 		time.setTimeFormat(Setting.value('timeFormat'));
```

Comment:
I don't think that's the right fix. The call to `this.applySettingsSideEffects()` in app.ts should already do this. And if it doesn't, that's where it should be fixed

Reply:
`applySettingsSideEffects()` does seem to already do this. Removing `time.setDateFormat` and `time.setTimeFormat` here seems to fix the issue as well.

Reply:
@laurent22, thanks for the review. Would you mind taking a look at my updates?

Reply:
Hmm, I looked at it but I really have no idea what will be the impact of the change. Unfortunately this part of the code has no unit tests so it's possible some subtle bug has been introduced.